roles/postgres/vars/main.yml

```diff
@@ -0,0 +1,14 @@
+postgresql_version: "14"
```

Comment:
How is 14 being installed? If it's just using apt, and we're on Debian stable, then the version would be 13.

If it depends on the Debian version, is there a nicer way to get the pg version than hard-coding it. Otherwise, upgrading Debian would entail remembering to update this variable too.

Reply:
hmm yea, we could either track the most recent version in debian, or we could add the postgres apt repo, which works similar to deadsnakes's ppa